bench/wtperf/config.c

```diff
@@ -765,11 +794,12 @@ config_sanity(WTPERF *wtperf)
 
     if (wtperf->workload != NULL)
         for (i = 0, workp = wtperf->workload; i < wtperf->workload_cnt; ++i, ++workp) {
-            if (opts->readonly &&
-              (workp->insert != 0 || workp->update != 0 || workp->truncate != 0)) {
+            if (opts->readonly && (workp->insert != 0 || workp->modify != 0 ||
+                                    workp->truncate != 0 || workp->update != 0)) {
```

Comment:
Whitespace. This should not be spaced all the way over there.

Reply:
This is because of new clang format, continuation indentation is applying after the nested brackets. I even tried to bring the entire second block down, still, the format is returning to the same way.